daemon/main.c

```diff
@@ -729,6 +730,8 @@ int main(int argc, char **argv) {
                 case 'D':
                     dont_fork = 1;
                     break;
+                case 'd':
+                    break;
```

Comment:
and:

```
dont_fork = 0;
````

So, to change later the default we can just change the default value of `dont_fork`.

Reply:
updated!